Library/Homebrew/extend/os/linux/hardware/cpu.rb

```diff
@@ -47,6 +47,8 @@ def family
             :haswell
           when 0x3d, 0x47, 0x4f, 0x56
             :broadwell
```

Comment:
Unfortunately, it looks like we're also missing Skylake. The only hardware I have to test with is my laptop, so there's nothing I can do about that.